### ISSUE TITLE
TextView text rendering problem - text is cut off

### ISSUE THREAD

**Issue by littledot** [google/flexbox-layout#39]
Hope I'm using Flexbox correctly, but I occasionally see TextView within a Flexbox render text incorrectly. 

![screenshot_2016-05-17-08-46-01](https://cloud.githubusercontent.com/assets/1559794/15329002/f3f1fc54-1c0b-11e6-86d6-beb25441fbc5.png)
Samsung S6; OS 5.1.1
![flexbox-bug](https://cloud.githubusercontent.com/assets/1559794/15329805/e173607e-1c0e-11e6-9b3c-7bd00afe39ca.png)
Emulator; OS 6

To repro, add this layout
```xml
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="wrap_content">

    <View
        android:layout_width="4dp"
        android:layout_height="match_parent"
        android:background="@color/primary1"/>

    <com.google.android.flexbox.FlexboxLayout
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:paddingBottom="4dp"
        android:paddingEnd="8dp"
        android:paddingLeft="8dp"
        android:paddingRight="8dp"
        android:paddingStart="8dp"
        android:paddingTop="4dp"
        app:flexWrap="wrap">

        <TextView
            android:id="@+id/content"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginTop="8dp"
            android:textSize="14sp"
            app:layout_flexBasisPercent="100%"/>

    </com.google.android.flexbox.FlexboxLayout>
</LinearLayout>
```
and Activity
```java
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.comment_item_layout);
        TextView tv3 = (TextView) findViewById(R.id.content);
        tv3.setText(Html.fromHtml("<p>Since we&#x27;re on the subject of hard drives, I hope this will be helpful or " +
                "interesting:<p>It has been our experience that it is no longer possible to buy new, non-fraudulant " +
                "drives of <i>a sort of recent, but not brand new</i> drive model from Amazon.<p>So, for instance, in" +
                " early 2016 if you want to buy 4TB enterprise drives from Amazon, you will find them, they will be " +
                "classified as brand new, and they will be sold by some big amazon parts seller.<p>When you receive " +
                "them they will be nice and shiny brand new wrapped - perfectly sealed - and when you spin them up, " +
                "SMART stats will show 4000-6000 hours of use and that they are 2-3 years old.<p>This is almost " +
                "universal and has been happening for at least 3-4 years. These sellers are selling the drives as " +
                "brand new and they are anything but. When you complain, they will immediately exchange or refund - " +
                "there&#x27;s never a hassle there - and once in a while the seller will spout some bullshit about " +
                "the drives being &quot;new pulls&quot; ... that is, drives they stripped from unsold servers&#x2F;" +
                "desktops.<p>Hope that helps."));
    }
}
```
Support Lib Version: 23.3.0; CompileSDK 23; TargetSDK 22

**Comment by thagikura** [comment]
Thanks for reporting. I'll have a look  to see if it's a bug with the FlexboxLayout

**Comment by thagikura** [comment]
I think this issue was caused by #36.
If so, it should be fixed by the 0.1.3 version of the library.

I was able to reproduce the issue with the 0.1.2 version and verified that the issue was fixed with the 0.1.3 version of the library.
Closing this. If it still persists, please re-open it.
